extensions/ui/org.eclipse.smarthome.ui.paper/tests/configuration.binding.test.js

```diff
@@ -0,0 +1,71 @@
+describe('module PaperUI.controllers.configuration.bindings', function() {
+    beforeEach(function() {
+        module('PaperUI');
+    });
+    var restConfig;
```

Comment:
Are you sure? Doesn't this run before all tests so we don't need to explicitly call it before the test, right?